bin/keys.py

```diff
@@ -36,7 +36,8 @@ def generateSerialKey(length):
         serialKey += random.choice(characters)
     return serialKey
 
-# Data Format: licensekey-hardwareID 
+# Data Format: licensekey:hardwareID 
+#TODO - CATCH ERROR IF DECRYPTION FAILS. SERVER CRASHES
```

Comment:
The Server does not crash; only the worker does. This essentially means that the server would still be able to take in requests, but the specific request that experienced this fatal error would receive an 500 HTTP Code as a return.

A fix will be implemented this Tuesday (September, 13)

Reply:
Fixed in the new merge request.